### ISSUE TITLE
clearing blocked nodes fails with

### ISSUE THREAD

**Issue by xmonader** [threefoldtech/js-sdk#1157]
to reproduce 

```
JS-NG> j.sals.reservation_chatflow.reservation_chatflow.list_blocked_nodes()      
{}

JS-NG>  j.sals.reservation_chatflow.reservation_chatflow.clear_blocked_nodes() 
```

# stacktrace
```
➜  js-sdk git:(development) ✗ poetry run jsng
JS-NG> j.sals.reservation_chatflow.reservation_chatflow.clear_blocked_nodes()     
2020-09-14 14:49:06.250 | ERROR    | jumpscale.tools.errorhandler.errorhandler:_handle_exception:55 - redis.exceptions.DataError: Invalid input of type: 'list'. Convert to a bytes, string, int or float first.
Traceback (most recent call last):
  File "/home/xmonader/wspace/js/js-sdk/.venv/bin/jsng", line 8, in <module>
    sys.exit(run())
    │   │    └ <Command run>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
           │    │     │       └ {}
           │    │     └ ()
           │    └ <function BaseCommand.main at 0x7ff8be1161f0>
           └ <Command run>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
         │    │      └ <click.core.Context object at 0x7ff8be678430>
         │    └ <function Command.invoke at 0x7ff8be116b80>
         └ <Command run>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           │   │      │    │           │   └ {'command': None}
           │   │      │    │           └ <click.core.Context object at 0x7ff8be678430>
           │   │      │    └ <function run at 0x7ff8be0b0f70>
           │   │      └ <Command run>
           │   └ <function Context.invoke at 0x7ff8be115ca0>
           └ <click.core.Context object at 0x7ff8be678430>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
           │         │       └ {'command': None}
           │         └ ()
           └ <function run at 0x7ff8be0b0f70>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/jumpscale/entry_points/jsng.py", line 30, in run
    sys.exit(embed(globals(), locals(), configure=ptconfig, history_filename=HISTORY_FILENAME))
    │   │    │                                    │                          └ '/home/xmonader/.jsng/history.txt'
    │   │    │                                    └ <function ptconfig at 0x7ff8bd626ca0>
    │   │    └ <function embed at 0x7ff8bd4cac10>
    │   └ <built-in function exit>
    └ <module 'sys' (built-in)>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/ptpython/repl.py", line 350, in embed
    repl.run()
    │    └ <function PythonRepl.run at 0x7ff8bd4ca790>
    └ <ptpython.repl.PythonRepl object at 0x7ff8bd670dc0>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/ptpython/repl.py", line 74, in run
    self._process_text(text)
    │    │             └ 'j.sals.reservation_chatflow.reservation_chatflow.clear_blocked_nodes()'
    │    └ <function PythonRepl._process_text at 0x7ff8bd4ca820>
    └ <ptpython.repl.PythonRepl object at 0x7ff8bd670dc0>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/ptpython/repl.py", line 88, in _process_text
    self._handle_exception(e)
    │    │                 └ DataError("Invalid input of type: 'list'. Convert to a bytes, string, int or float first.")
    │    └ functools.partial(<function patched_handle_exception at 0x7ff8be0baa60>, <ptpython.repl.PythonRepl object at 0x7ff8bd670dc0>)
    └ <ptpython.repl.PythonRepl object at 0x7ff8bd670dc0>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/ptpython/repl.py", line 163, in _execute
    six.exec_(code, self.get_globals(), self.get_locals())
    │   │     │     │    │              │    └ <function embed.<locals>.get_locals at 0x7ff8bd4caca0>
    │   │     │     │    │              └ <ptpython.repl.PythonRepl object at 0x7ff8bd670dc0>
    │   │     │     │    └ <function embed.<locals>.get_globals at 0x7ff8be0ad5e0>
    │   │     │     └ <ptpython.repl.PythonRepl object at 0x7ff8bd670dc0>
    │   │     └ <code object <module> at 0x7ff8b63f5500, file "<stdin>", line 1>
    │   └ <built-in function exec>
    └ <module 'six' from '/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/six.py'>
> File "<stdin>", line 1, in <module>

  File "/home/xmonader/wspace/js/js-sdk/jumpscale/sals/reservation_chatflow/reservation_chatflow.py", line 1630, in clear_blocked_nodes
    j.core.db.delete(blocked_node_keys)
    │ │              └ []
    │ └ <property object at 0x7ff8be136e00>
    └ <jumpscale.loader.J object at 0x7ff8be128640>

  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/redis/client.py", line 1567, in delete
    return self.execute_command('DEL', *names)
           │    │                       └ ([],)
           │    └ <function Redis.execute_command at 0x7ff8bc586af0>
           └ Redis<ConnectionPool<Connection<host=localhost,port=6379,db=0>>>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/redis/client.py", line 900, in execute_command
    conn.send_command(*args)
    │    │             └ ('DEL', [])
    │    └ <function Connection.send_command at 0x7ff8bc5644c0>
    └ Connection<host=localhost,port=6379,db=0>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/redis/connection.py", line 725, in send_command
    self.send_packed_command(self.pack_command(*args),
    │    │                   │    │             └ ('DEL', [])
    │    │                   │    └ <function Connection.pack_command at 0x7ff8bc564670>
    │    │                   └ Connection<host=localhost,port=6379,db=0>
    │    └ <function Connection.send_packed_command at 0x7ff8bc564430>
    └ Connection<host=localhost,port=6379,db=0>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/redis/connection.py", line 775, in pack_command
    for arg in imap(self.encoder.encode, args):
        │      │    │    │       │       └ (b'DEL', [])
        │      │    │    │       └ <function Encoder.encode at 0x7ff8bc5c7f70>
        │      │    │    └ <redis.connection.Encoder object at 0x7ff8b6dc2d30>
        │      │    └ Connection<host=localhost,port=6379,db=0>
        │      └ <class 'map'>
        └ b'DEL'
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/redis/connection.py", line 119, in encode
    raise DataError("Invalid input of type: '%s'. Convert to a "
          └ <class 'redis.exceptions.DataError'>

redis.exceptions.DataError: Invalid input of type: 'list'. Convert to a bytes, string, int or float first.
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/xmonader/wspace/js/js-sdk/jumpscale/sals/reservation_chatflow/reservation_chatflow.py", line 1630, in clear_blocked_nodes
    j.core.db.delete(blocked_node_keys)
    │                └ []
    └ <jumpscale.loader.J object at 0x7ff8be128640>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/redis/client.py", line 1567, in delete
    return self.execute_command('DEL', *names)
           │                            └ ([],)
           └ Redis<ConnectionPool<Connection<host=localhost,port=6379,db=0>>>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/redis/client.py", line 900, in execute_command
    conn.send_command(*args)
    │                  └ ('DEL', [])
    └ Connection<host=localhost,port=6379,db=0>
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/redis/connection.py", line 725, in send_command
    self.send_packed_command(self.pack_command(*args),
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/redis/connection.py", line 775, in pack_command
    for arg in imap(self.encoder.encode, args):
  File "/home/xmonader/wspace/js/js-sdk/.venv/lib/python3.8/site-packages/redis/connection.py", line 119, in encode
    raise DataError("Invalid input of type: '%s'. Convert to a "
redis.exceptions.DataError: Invalid input of type: 'list'. Convert to a bytes, string, int or float first.

Invalid input of type: 'list'. Convert to a bytes, string, int or float first.
JS-NG>  
```

**Comment by xmonader** [comment]
should be fixed in #1159

**Comment by Dina-Abd-Elrahman** [comment]
**Commit_ID**  bd546540e3930ce038ab925ef8c20011a65818fd , **Branch** development

![ff](https://user-images.githubusercontent.com/21218053/93180518-94353b80-f737-11ea-8654-d1f8aab8ee6b.png)
